web/app/templates/base.html

```diff
@@ -30,6 +31,16 @@
   {{ moment.include_moment() }}
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
   <script type="text/javascript">
+  
     var $SCRIPT_ROOT = {{ url_for('main.index', _external = True)|tojson|safe }}
+    var csrftoken = $('meta[name=csrf-token]').attr('content')
```

Comment:
And where should that JS come from? We use a few major js libraries that we, if we're really obnoxious about it, can serve from our server.

The users have no way of entering code (we don't have that kind of data) so there's no risk of us serving it. Web browsers are sandboxed to hell today so the only scenario left is a completely hijacked computer or a malicious user. There's no way to detect that for us and IMHO far beyond what's reasonable to look for.

This piece of code is as good as it gets.